src/module-rebuilder.ts

```diff
@@ -120,25 +120,26 @@ export class ModuleRebuilder {
     const buildLocation = path.resolve(this.modulePath, 'build', this.rebuilder.buildType);
 
     d('searching for .node file', buildLocation);
-    const buildLocationFiles = await fs.readdir(buildLocation);
+    const buildLocationFiles = await fs.promises.readdir(buildLocation);
```

Comment:
Since this is a `graceful-fs` call, it'll need to be passed through `util.promisify`